crates/holochain_state/src/mutations.rs

```diff
@@ -311,7 +311,7 @@ pub fn set_validation_stage(
         ValidationLimboStatus::AwaitingAppDeps(_) => Some(2),
         ValidationLimboStatus::AwaitingIntegration => Some(3),
     };
-    let now = holochain_types::timestamp::now().0;
+    let now = holochain_types::timestamp::now().secs();
```

Comment:
```suggestion
    let now = holochain_types::timestamp::now();
```

Reply:
This is wrong. It should be the actual timestamp not just the seconds here.